docs/articles/faq/README.md

```diff
@@ -0,0 +1,253 @@
+
+Historically, TestCafe first appeared as a paid standalone tool. Beside the test runner,
+it featured the [Control Panel](https://testcafe.devexpress.com/Documentation/Using_TestCafe/Control_Panel/),
+a visual interface for managing your tests (creating, modifying, running), and an innovative
+[Visual Test Recorder](https://testcafe.devexpress.com/Documentation/Using_TestCafe/Visual_Test_Recorder/)
+that allowed you to record tests by pointing and clicking through the test scenario in the browser.
+
+This version was actively developed until 2015 when we decided to release its core as an open-source project.
+The last major release of the paid version happened in the summer 2015, which was v15.1.
+After that, it was switched to a maintenance-only mode. The website at [https://testcafe.devexpress.com](https://testcafe.devexpress.com)
+currently documents the latest paid version.
+
+In late 2016, we have released an open-source TestCafe version under the same name – just ***TestCafe***.
+This version contains a lot of new functionality and many important changes compared to the core of the last paid version.
+
+* it has a completely new API and offers a new approach to writing tests;
+* it has no UI to manage and run tests. You can run test from the CLI or from a node.js module;
+* it's more convenient to integrate into node.js development workflow
+* it has some new features (like es6 support, flexible selectors and smart assertions, authentication features etc.)
+
+You can learn more about the open-source version at [https://devexpress.github.io/testcafe](https://devexpress.github.io/testcafe).
+
+However, we didn't abandon our aspirations to create a competitive proprietary product.
+We plan to release a completely new commercial version based on the revised open-source TestCafe.
+Stay tuned to [https://testcafe.devexpress.com](https://testcafe.devexpress.com) to keep in touch
+about the new product that will be called ***TestCafe Studio***.
+
+### Which browsers does TestCafe support? What are the exact supported versions?
+
+We test TestCafe against a limited number of browsers. We call these browsers *officially* supported.
+You can find the list of these browsers in [our documentation](../documentation/using-testcafe/common-concepts/browser-support.md).
+
+At the same time, TestCafe should work with almost every modern browser with extensive support for HTML5.
+You can check if TestCafe works in the browser of your interest by yourself.
+
+As to the exact browser versions, we test TestCafe against two newest versions
+of each officially supported browser (unless the exact version is specified explicitly
+in the supported browser list). However, TestCafe actually works with many older versions
+since we usually refrain from using cutting edge features in TestCafe code. In fact,
+TestCafe should work in any officially supported browser released in the last three years.
+
```

Comment:
TestCafe works with two latest versions of oficially supported browsers. Exceptions are specified explicitly in the supported browsers list. However, TestCafe usually works in any version of such browsers released in the last three years.